go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1 // indirect
 	github.com/hashicorp/go-retryablehttp v0.7.1 // indirect
 	github.com/hashicorp/go-slug v0.8.1
-	github.com/hashicorp/go-tfe v1.2.0
+	github.com/hashicorp/go-tfe v1.2.1-0.20220607193402-515f37b540ef
```

Comment:
Does this need to be updated or is this version tag correct?

Reply:
Yes, this is what was holding this up! I was waiting on the 1.3.0 release